src/components/Landing.js

```diff
@@ -40,6 +42,14 @@ class Landing extends React.Component {
     window.ga('send', 'pageview', window.location.pathname);
   }
 
+  handleLetsPlayButtonClick(e) {
```

Comment:
No need for this function if we update the parent's state from the other handlers.

Reply:
Well we currently don't... see my earlier comment, and tell me what you think.